Work around confusing import of cpplint for Python 3.11

We have a directory named cpplint and file named cpplint/cpplint.py.

We want to import the latter, but sometimes we end up with the empty
__init__.py from the former. Force the issue by munging Python's path.

--- cpplint/cpplint_unittest.py
+++ cpplint/cpplint_unittest.py
@@ -41,6 +41,7 @@
 import sys
 import unittest
 
+sys.path.insert(0, os.path.dirname(os.path.abspath(__file__)))
 import cpplint
 
 try:
   xrange          # Python 2